src/static/js/libs/gamepad.js

```diff
@@ -562,12 +562,12 @@
     if (this.gamepads.length > 0) {
       this._fire(Gamepad.Event.TICK, this.gamepads);
     }
-    //window.requestAnimationFrame(function() {
-    //	self._update();
-    //});
-    setTimeout(function () {
-      self._update();
-    }, 100);
+    window.requestAnimationFrame(function() {
```

Comment:
To expand on this, I will eventually be updating the control update loop to operate at 60hz. 100ms of control latency essentially creates a worst case of (100ms + video latency ) perceived latency, so if the video latency is 110ms, the worst case perceived latency is 210ms during periods of consistent control input deltas (such as when trying to perform a dynamic maneuver).

The gamepad processing code should be extremely light. If there are no new gamepad events, there should be no further processing to do. We should design for this sooner rather than later and accept the processing requirements that this entails.

This also creates a worst case latency of 100ms for any other button based command in the system, such as toggling lights, recording, etc.